grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/fragment/OutShortcutFragment.java

```diff
@@ -123,4 +157,9 @@ public int hashCode() {
         result = 31 * result + relationTypeLabels.hashCode();
         return result;
     }
+
+    @Override
+    public boolean operatesOnEdge() {
+        return true;
```

Comment:
I sense having this be defaulted to true may come back to bite us. I would rather have `throw UnsupportedOperation` or something like that.